pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -291,6 +292,10 @@ func buildHTTPVirtualServices(
 			vs.Route = route
 		}
 
+		if r.Timeouts != nil && r.Timeouts.Request != nil {
+			vs.Timeout = durationpb.New(r.Timeouts.Request.Duration)
```

Comment:
When should we use `BackendRequest`? Looks it is outbound time, which should be set to vs timeout

Reply:
`BackendRequest` maps to VS `rules.retires.perTryTimeout`. Here is the draft GEP this is implementing: https://github.com/kubernetes-sigs/gateway-api/blob/ad0e9c7a569bde169254a47a68e7f760ad6ebb15/geps/gep-1742.md#api

Reply:
so we can support BackendRequest, right? we just don't in this PR, but we can